geolocator_android/lib/src/types/android_position.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:geolocator_platform_interface/geolocator_platform_interface.dart';
+// ignore: depend_on_referenced_packages
+import 'package:meta/meta.dart';
+
+/// Contains additional location information only available on Android platforms.
+@immutable
+class AndroidPosition extends Position {
+  /// Constructs an instance with the given values for testing. [AndroidPosition]
+  /// instances constructed this way won't actually reflect any real information
+  /// from the platform, just whatever was passed in at construction time.
+  const AndroidPosition(
+      {required this.satelliteCount,
```

Comment:
nit: small formatting mistake:

```suggestion
  const AndroidPosition({
      required this.satelliteCount,
```